### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -21648,7 +21666,13 @@ sitautions->situations
 sitck->stick
 siteu->site
 sitll->still
+situationals->situations, situational,
+situationly->situational
+situationnal->situational
+situatuion->situation
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Add `situatuions->situations`

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -21648,7 +21666,13 @@ sitautions->situations
 sitck->stick
 siteu->site
 sitll->still
+situationals->situations, situational,
+situationly->situational
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Suggest `situationally` first

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -21075,6 +21090,9 @@ segemnts->segments
 segfualt->segfault
 segfualts->segfaults
 segmantation->segmentation
+segmend->segment
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Add `segmended->segmented`

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -14752,6 +14765,7 @@ materaials->materials
 materail->material
 materails->materials
 materalists->materialist
+materiasl->materials
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Also suggest `material`

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -11937,6 +11945,7 @@ higeine->hygiene
 higer->higher
 higest->highest
 high-affort->high-effort
+highe->high, higher,
 highes->highest
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: I know it's not your change, but could you also add `highs` to this as a suggestion.

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -4652,6 +4653,7 @@ collegues->colleagues
 collektion->collection
 colletion->collection
 collison->collision, collusion,
+collisons->collisions
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Can you suggest `collusion` AND `collusions` please.

**Comment by luzpaz** [comment]
done

**Pull request merged**